xla/service/gpu/BUILD

```diff
@@ -3616,6 +3616,7 @@ cc_library(
         "@com_google_absl//absl/types:span",
         "//xla:shape_util",
         "//xla/hlo/ir:hlo_module_group",
+        "//xla/hlo/experimental/auto_sharding",
```

Comment:
Just a heads up that we made some additional changes internally and moved this dependency to a normal list (outside of `if_gpu_is_configured`).